### PR TITLE
Fix: README.md

### PR DESCRIPTION
**Pull request by PentesterPriyanshu**
Added Twitter and Discord badges to the text to provide quick access to the Hanko community. The badges will enhance user experience and enable easy navigation to the community platforms.

# Description

Added Twitter and Discord badges to the text to provide quick access to the Hanko community. The badges will enhance user experience and enable easy navigation to the community platforms.

# Implementation

The badges were added directly to the existing text, utilizing Markdown syntax. The purpose is to make the community platforms easily accessible with a single click.

# Tests

To verify the changes, follow these steps:

1. Open the relevant page where the Hanko community text is located.
2. Verify that the Twitter and Discord badges are visible and clickable.
3. Confirm that clicking the badges redirects users to the respective Hanko community pages.

# Todos

None.

# Additional context

The addition of the badges aims to create an eye-catching and user-friendly experience, facilitating a seamless transition to the Hanko community platforms with just one click.
Screenshot from 2023-10-31 00-06-02
Screenshot from 2023-10-31 00-06-25

### CODE REVIEW

**Comment by PentesterPriyanshu** [comment]
@FlxMgdnz  look at this changed readme.md i have added X badge with discord community badge also

**Comment by PentesterPriyanshu** [comment]
@lfleischmann  see this i have added badges of X  and discord now people will easily go to our community because badges are eye catchy

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 [![GoDoc](https://godoc.org/github.com/teamhanko/hanko?status.svg)](https://godoc.org/github.com/teamhanko/hanko)
 [![npm (scoped)](https://img.shields.io/npm/v/@teamhanko/hanko-elements?label=hanko-elements)](https://www.npmjs.com/package/@teamhanko/hanko-elements)
 [![npm (scoped)](https://img.shields.io/npm/v/@teamhanko/hanko-frontend-sdk?label=hanko-frontend-sdk)](https://www.npmjs.com/package/@teamhanko/hanko-frontend-sdk)
+<img src="https://img.shields.io/static/v1?label=Join&message=Discord&color=blueviolet&style=flat-square" alt="Join Discord">
+<img src="https://img.shields.io/twitter/follow/hanko_io?style=social" alt="Follow us on Twitter">
```
Comment (FlxMgdnz) [inline README.md:14]: Links are missing

File: README.md [hunk 2]
```diff
@@ -112,5 +114,15 @@ If you have any questions or issues, please check this project's [Q&A section in
...
 We invite you to join our growing [Discord community](https://www.hanko.io/community) if you want to get the latest updates on passkeys, WebAuthn, and this project or if you just want to chat with us. You can also [follow us on Twitter](https://x.com/hanko_io).
 
+<p align="center">
+    <a href="https://www.hanko.io/community">
+        <img src="https://img.shields.io/static/v1?label=Join&message=Discord&color=blueviolet&style=flat-square" alt="Join Discord">
+    </a>
+    <a href="https://x.com/hanko_io">
+        <img src="https://img.shields.io/twitter/follow/hanko_io?style=social" alt="Follow us on Twitter">
+    </a>
+</p>
```
Comment (FlxMgdnz) [inline README.md:124]: I'd prefer left-aligned buttons (like the cal.com button above)

**Review by FlxMgdnz — CHANGES_REQUESTED** [review]

**Comment by PentesterPriyanshu** [comment]
i am resolving all these just wait dont merge any similiar pr

File: README.md (hunk 1 above)
Comment (PentesterPriyanshu) [reply README.md]: all issues resolved mentioned by you ,thank you

File: README.md (hunk 2 above)
Comment (PentesterPriyanshu) [reply README.md]: all issue resolved mentioned by you,thankyou

**Comment by PentesterPriyanshu** [comment]
@FlxMgdnz now review and merge it , thank you

**Pull request closed without merging**